pkg/storaged/format-dialog.jsx

```diff
@@ -158,6 +158,12 @@ function initial_tab_options(client, block) {
         if (utils.is_netdev(client, p)) {
             options["_netdev"] = true;
         }
+        // HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1589541
+        if (client.vdo_overlay.find_by_block(client.blocks[p])) {
+            options["_netdev"] = true;
+            options["x-systemd.device-timeout=0"] = true;
+            options["x-systemd.requires=vdo.service"] = true; // XXX - is this actually recognized in crypttab?
```

Comment:
It's not documented that this has any effect in crypttab. Could this get an integration test that validates that this ends up in `systemctl show -p Requires XXX.mount`?

Reply:
> It's not documented that this has any effect in crypttab.

I will figure out whether we need this dependency.  If yes, I'll keep this here and push for systemd to also respect it in crypttab (if it doesn't yet).

Reply:
> Could this get an integration test that validates that this ends up in systemctl show -p Requires XXX.mount

We could do that and if it doesn't, we can use a knownissue to track the necessary systemd change.

Reply:
> I will figure out whether we need this dependency.

We need it when vdo.service is not enabled, otherwsie _netdev is enough.  Also, only one entry in fstab or crypttab needs the dependency since it is enough to start vdo.service once.

So we should add that option into crypttab also and ask systemd to recognize it.  I'll get that rolling.

Reply:
>> Could this get an integration test that validates that this ends up in systemctl show -p Requires XXX.mount

> We could do that and if it doesn't, we can use a knownissue to track the necessary systemd change.

I skipped this for now just to reduce the scope if this.